plugin/src/main/kotlin/io/github/ttypic/swiftklib/gradle/templates/CreatePackageSwift.kt

```diff
@@ -1,9 +1,12 @@
 package io.github.ttypic.swiftklib.gradle.templates
 
+import io.github.ttypic.swiftklib.gradle.SwiftPackageDependency
+
 internal fun createPackageSwiftContents(
     cinteropName: String,
+    dependencies: Collection<SwiftPackageDependency>
 ): String = """
-    // swift-tools-version:5.5
+    // swift-tools-version:5.6
     import PackageDescription
 
     let package = Package(
```

Comment:
can you please add options to set "platforms" too in the Package. 
Pls take a quick look -> https://github.com/wh173d3v11/swift-klib-plugin/blob/2286d680ddfdc057e077e9306ad9e11cb54806f2/plugin/src/main/kotlin/io/github/ttypic/swiftklib/gradle/templates/CreatePackageSwift.kt#L11
Example - platforms: [.iOS(.v13), .macOS(.v11)]

Because, in some cases we need to set the minimum version. So, if i not add this line in my usecase, i got error like 
"the library 'iosOnnx' requires macos 10.13, but depends on the product 'onnxruntime' which requires macos 11.0; consider changing the library 'iosOnnx' to require macos 11.0 or later, or the product 'onnxruntime' to require macos 10.13 or earlier.
"

Reply:
I'm still not sure about this whole Package.swift generation, tbh.
but another approach is even worse - parsing Package.swift ourselves 😅

Reply:
@IlyaGulya I feel the same way! But at least by providing an option for self-parsing, it adds flexibility, so others won’t need to go through the hassle I did—modifying and locally publishing changes to the plugin code just to make it work!

Reply:
it's possible to do some operation on the manifest with the command line see: https://github.com/ttypic/swift-klib-plugin/issues/37#issuecomment-2453394271